test/oatpp/core/data/stream/BufferStreamTest.cpp

```diff
@@ -47,11 +47,11 @@ void BufferStreamTest::onRun() {
     OATPP_ASSERT(stream.toString() == oatpp::utils::conversion::int32ToStr(101));
 
     stream.setCurrentPosition(0);
-    stream << (v_float32)101.1;
+    stream << static_cast<v_float32>(101.1);
     OATPP_ASSERT(stream.toString() == oatpp::utils::conversion::float32ToStr(101.1f));
 
     stream.setCurrentPosition(0);
-    stream << (v_float64)101.1;
+    stream << 101.1;
```

Comment:
Please do not remove this explicit type cast.
It may be run on a different platform where ex.: FP literal type is float32 by default

Reply:
done